src/forest.nml

```diff
@@ -1,37 +1,443 @@
+	xoffset: 8;
+	yoffset: 0;
+	xextent: 8;
+	yextent: 8;
+	}
+	//right corner
+	building {
+	sprite: (terrain_type == TILETYPE_SNOW ? (construction_state==0 ? 1772 : construction_state==1 ? 1773 : construction_state==2 ? 1774 : 1775) : (construction_state==0 ? 1716 : construction_state==1 ? 1717 : construction_state==2 ? 1718 : 1719));
+	xoffset: 0;
+	yoffset: 8;
+	xextent: 8;
+	yextent: 8;
+	}
+}
+
+spritelayout sprlay_forest_snowy_mixed_growing{
+	ground	{sprite: (terrain_type == TILETYPE_SNOW ? 4531 : 3962);}
+	//top corner 
+	building	{
+	sprite: (terrain_type == TILETYPE_SNOW ? (construction_state==0 ? 1772 : construction_state==1 ? 1773 : construction_state==2 ? 1774 : 1775) : (construction_state==0 ? 1716 : construction_state==1 ? 1717 : construction_state==2 ? 1718 : 1719));
+	xoffset: 0;
+	yoffset: 0;
+	xextent: 8;
+	yextent: 8;
+	}
+	//bottom corner
+	building {
+	sprite: (terrain_type == TILETYPE_SNOW ? (construction_state==0 ? 1772 : construction_state==1 ? 1773 : construction_state==2 ? 1774 : 1775) : (construction_state==0 ? 1716 : construction_state==1 ? 1717 : construction_state==2 ? 1718 : 1719));
+	xoffset: 8;
+	yoffset: 8;
+	xextent: 8;
+	yextent: 8;
+	}
+	// left corner
+	building {
+	sprite: (terrain_type == TILETYPE_SNOW ? (construction_state==0 ? 1765 : construction_state==1 ? 1766 : construction_state==2 ? 1767 : 1768) : (construction_state==0 ? 1709 : construction_state==1 ? 1710 : construction_state==2 ? 1711 : 1712));
+	xoffset: 8;
+	yoffset: 0;
+	xextent: 8;
+	yextent: 8;
+	}
+	//right corner
+	building {
+	sprite: (terrain_type == TILETYPE_SNOW ? (construction_state==0 ? 1765 : construction_state==1 ? 1766 : construction_state==2 ? 1767 : 1768) : (construction_state==0 ? 1709 : construction_state==1 ? 1710 : construction_state==2 ? 1711 : 1712));
+	xoffset: 0;
+	yoffset: 8;
+	xextent: 8;
+	yextent: 8;
+	}
+}
+
+/* TEMPERATE and ARCTIC graphics switches */
+
+switch (FEAT_INDUSTRYTILES, SELF, switch_forest_pine1_selector, climate == CLIMATE_ARCTIC){
+	1: return sprlay_forest_snowy_pine1_growing;
+	0: return sprlay_forest_temp_pine1_growing;
+	}
+	
+switch (FEAT_INDUSTRYTILES, SELF, switch_forest_pine2_selector, climate == CLIMATE_ARCTIC){
+	1: return sprlay_forest_snowy_pine2_growing;
+	0: return sprlay_forest_temp_pine2_growing;
+	}
+switch (FEAT_INDUSTRYTILES, SELF, switch_forest_brdlfs_selector, climate == CLIMATE_ARCTIC){
+	1: return sprlay_forest_snowy_brdlfs_growing;
+	0: return sprlay_forest_temp_brdlfs_growing;
+	}
+	
+switch (FEAT_INDUSTRYTILES, SELF, switch_forest_mixed_selector, climate == CLIMATE_ARCTIC){
+	1: return sprlay_forest_snowy_mixed_growing;
+	0: return sprlay_forest_temp_mixed_growing;
+	}	
+
+item (FEAT_INDUSTRYTILES, ind_tile_forest_mixed) {
+    property {
+        substitute: 16;
+        //override: 16;
```

Comment:
If this isn't used, it should be removed instead of commented out. (Here and below).

Reply:
This I wasn't sure I needed to delete completely, actually. I understand the substitute keyword function, but override  keyword? Is it really needed in case of all new industry tiles and industry? Please advise.

Reply:
I commented it out early on when I was figuring stuff out and then forgot to add back.

It is now not commented in all related objects.